app/controllers/session_controller.rb

```diff
@@ -332,6 +333,34 @@ def create
     end
   end
 
+  def passkey_challenge
+    render json: DiscourseWebauthn.stage_challenge(current_user, secure_session)
+  end
+
+  def passkey_auth_perform
```

Comment:
What do you think about `login_with_passkey` instead? It'll match the other controller methods more closely like `login`, `login_with_sso` etc.

Reply:
Sure, that's better, thanks!